### ISSUE TITLE
Take book covers and thumbnails out of the taborder on results & selections lists

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#338]
On results & selections lists, for all document thumbnails:
- the image should have alt=""  (currently "Book cover")
- the link should have tabindex=-1  (that's "minus one") to remove it from the tab order.

Outcome:  Keyboard user will have one tab per title link (good). Screen reader will not say "Book cover" for every item (good). Links list will still have two links per item, but one will be blank (not super great).
